main.py

```diff
@@ -106,7 +113,15 @@ def loop(self):
             # Draw contours
             target.draw_contours(filtered_contours, contour_image)
             # Find distance and angle
-            distance, angle = target.measurements(original, filtered_contours)
+            distance, angle, cnt = target.measurements(original, filtered_contours)
+            # Get distance from realsense if applicable
+            if self.results.realsense:
+                x, y = utils.get_center(cnt)
+                if x:

```

Comment:
I guess you're checking if `x` is `None`, but this condition would be met if `x` is `0` also, so maybe you should change this.
```suggestion
                if x != 0:
```